framework/test/functional/rpc-api/ipc/ipc_client_in_memory.spec.ts

```diff
@@ -103,6 +103,14 @@ describe('api client ipc mode', () => {
 				client.invoke('app:getAccount', { address: 'randomString*&&^%^' }),
 			).rejects.toThrow('Specified key accounts:address: does not exist');
 		});
+
+		it('should return a list of registered actions', async () => {
+			const actions = await await client.invoke('app:getRegisteredActions');
+			expect(actions).toBeArray();
+			expect(actions).toContain('app:getConnectedPeers');
+			expect(actions).toContain('dpos:getAllDelegates');
+			expect(actions).toContain('hello:callGreet');
```

Comment:
Either we match exact array or match the length of array as well to be sure there are only these actions available.